src/debug/ee/arm64/arm64walker.cpp

```diff
@@ -366,7 +428,8 @@ BOOL  NativeWalker::DecodePCRelativeBranchInst(PT_CONTEXT context, const PRD_TYP
     _ASSERTE(walk == incomingwalk);
     return FALSE;
 }
-BOOL  NativeWalker::DecodeCallInst(PT_CONTEXT context, const PRD_TYPE& opcode, int& RegNum, WALK_TYPE& walk)
+
+BOOL  NativeWalker::DecodeCallInst(const PRD_TYPE& opcode, int& RegNum, WALK_TYPE& walk)
 {
     if ((opcode & 0xFF9FFC1F) == 0xD61F0000) //Unconditional Branch (register) instructions
```

Comment:
This is the BR <Xn> instruction, so the comment is clearer if is reads:

//  BR <Xn>  - Unconditional Branch (register) instructions